zeppelin-web/pom.xml

```diff
@@ -120,34 +152,77 @@
           </execution>
 
           <execution>
-            <id>yarn install</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>install --no-lockfile</arguments>
-            </configuration>
+              <id>yarn add bower</id>
+              <goals>
+                  <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <!--ignore-scripts-->
+                  <arguments>add bower --no-lockfile --no-bin-links</arguments>
```

Comment:
@Leemoonsoo this is your words:
>Like i mentioned above, run 'yarn add xx' changes 'package.json', which is version controlled >file. And many section in 'pom.xml' is duplicated with 'package.json'.

>How about find alternative way to solve the problem without touching package.json and >without duplicating the feature in two different places?

I have removed "postinstall" from package.json because it is triggered after each
```
yarn add
```
and fail build.
If I run 
```
$ yarn install
``` 
from command line this also failed (duplicates for ZEPPELIN-1928)
https://issues.apache.org/jira/browse/ZEPPELIN-1019
https://issues.apache.org/jira/browse/ZEPPELIN-1474
https://issues.apache.org/jira/browse/ZEPPELIN-380